lms/djangoapps/bookmarks/api.py

```diff
@@ -0,0 +1,27 @@
+"""
+This File contains bookmark api method(s).
+"""
+
+from bookmarks.serializers import BookmarkSerializer
+from bookmarks.models import Bookmark
+
+
+def get_bookmark(user, usage_key, fields=None, serialized=False):
+    """
+    Return bookmark model or data.
+
+    Args:
+        user (User): The user requesting the bookmark.
+        fields_to_add (list): List of fields to return for a bookmark.
+        usage_key (UsageKey): The usage id of an Xblock.
+        serialized (Bool): Decides to return object or json.
```

Comment:
State the default value (False), and clarify that if serialized is True, a JSON serialized version of the object is returned.